.github/workflows/CI.yml

```diff
@@ -84,6 +84,21 @@ jobs:
       run: |
         CHPL_HOME=$PWD make run-dyno-linters
 
+  make_check_llvm_none:
```

Comment:
It would be nice to declare this right next to the usual `make_check` job, since there is a lot of duplicate code between the two we don't want to forget to update together.